components/Navbar.js

```diff
@@ -17,8 +17,8 @@ export default function Navbar() {
 
       <footer>
         <StyledList>
-          <StyledDiv active={pathname === "/" ? true : false}>
-            <StyledLink href="/">
+          <StyledDiv active={pathname === "/home" ? true : false}>
```

Comment:
Do you use the StyledDiv component only to style the StyledLink component?

Reply:
I use the StyledDiv to highlight the active page icon